pyodide-build/pyodide_build/tests/test_buildall.py

```diff
@@ -6,7 +6,7 @@
 from pyodide_build import buildall
 import pytest
 
-PACKAGES_DIR = (Path(__file__) / ".." / ".." / ".." / "packages").resolve()
+PACKAGES_DIR = (Path(__file__).parents[3] / "packages").resolve()
```

Comment:
Maybe:
```suggestion
PACKAGES_DIR = (Path(__file__).absolute().parents[3] / "packages").resolve()
```
I think we should do this everywhere, if you try to run the file individually it won't work otherwise.

Reply:
It's actually already done with `resolve` at the end, absolute is not really public API https://bugs.python.org/issue29688